dev/system-test/firestore.ts

```diff
@@ -153,7 +154,7 @@ describe('Firestore class', () => {
       })
       .then(() => Promise.reject('set() should have failed'))
       .catch(err => {
-        expect(err).to.equal('The client has already been terminated');
+        expect(err).to.equal(CLIENT_TERMINATED_ERROR_MSG);
```

Comment:
FWIW Having the actual message here makes typos much more apparent.

Optional.

Reply:
done.